### PR TITLE
feat(TMRX-1260): update refactor title bar component

### PR DESCRIPTION
**Pull request by ibrahim-kurhan**
User Story

As a web reader
I want to see articles presented in this new slice
So that I can enjoy the most up to date reader experience

[TMRX-1260](https://nidigitalsolutions.jira.com/browse/TMRX-1260)

Acceptance Criteria

Update ContentBucket1 slice in Storybook using new tokens and presets, from new theme (v0.1.1) wherever possible, using the GridLayout.

Update cards, divider, and title bar in Storybook that feed into this slice, and for the ContentBucket2 slice.

Design approved by @Himesh Patel @Scott.Jones 

Design handoff: https://www.figma.com/file/phASjaJaTK2CD5MwZUZlZr/Slices-handoff?node-id=1-31 - Connect your Figma account 

Definition of Done

Code complete

Code reviewed/PR approved

Design reviewed

Merged into MASTER

Available to view in Storybook.

Examples of the different sub-themes applied are here showing the colour token values changing:

https://www.figma.com/file/phASjaJaTK2CD5MwZUZlZr/Slices-handoff?node-id=1%3A30&t=JUNRMpPpCiiwwiO1-1 - Connect your Figma account 

Please review this with @Himesh Patel & @Valeriia.Bielozorova next week for design reviews, and any questions.

Description

Build the UI for the new slice as per the design file.

Integration with TPA slice data to follow as a separate piece of work

Design handoff: https://www.figma.com/file/phASjaJaTK2CD5MwZUZlZr/Slices-handoff?node-id=1-31 - Connect your Figma account  

[TMRX-1260]: https://nidigitalsolutions.jira.com/browse/TMRX-1260?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ts-newskit/src/assets/ChevronRightIcon.tsx [hunk 1]
```diff
@@ -3,11 +3,11 @@ import * as React from 'react';
 const ChevronRigthIcon = (props: any) => (
```
Comment (douglasmik) [inline packages/ts-newskit/src/assets/ChevronRightIcon.tsx:3]: OBS: Typo
`ChevronRightIcon`

Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/assets/ChevronRightIcon.tsx:3]: ahh again:))

Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/assets/ChevronRightIcon.tsx]: fixed

File: packages/ts-newskit/src/components/slices/slice-header/index.tsx [hunk 2]
```diff
@@ -1,39 +1,42 @@
...
+export const SliceHeader = ({ title, href }: SliceHeaderProps) => {
   return (
-    <StyledStack flow="horizontal-center" stackDistribution="space-between">
-      <ColouredText
-        typographyPreset={{
-          xs: 'newPreset070',
-          lg: 'newPreset060'
-        }}
-        $color={color}
+    <Block stylePreset="sliceHeaderPreset">
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/slices/slice-header/index.tsx:12]: Just seen that there's a `Titlebar` component for this that might help
https://newskit.co.uk/components/title-bar/

Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/slices/slice-header/index.tsx:12]: changed

File: packages/ts-newskit/src/theme/times-web-light/style-presets/slice-header.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+export const sliceHeaderStylePresets = {
+  sliceHeaderPreset: {
+    base: {
+      borderStyle: 'dashed none none none',
```
[the remark this answers is not in the document]
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/theme/times-web-light/style-presets/slice-header.ts]: changed to borderTop: 'dashed'

**Review by douglasmik — APPROVED** [review]

**Pull request merged**